### DISCUSSION TITLE
1 file unhandled waring

### DISCUSSION THREAD

**Discussion by RMcGhee** [guoyingtao/Mantis#152] [Q&A] [answered]
Hey, thanks so much for this library, it really rocks! I have a problem that I just cannot seem to solve:

Showing All Messages
found 1 file(s) which are unhandled; explicitly declare them as resources or exclude from the target
/Project_Name/SourcePackages/checkouts/Mantis/Sources/Mantis/Info.plist

I've tried editing the Package.swift file to exclude this file in targets: [.target()], and the changes register in Xcode, but the warning doesn't go away. I feel like there's something obvious I'm missing, but Google-fu has failed me. Anyone have an idea of what to do here?

**Comment by RMcGhee** [comment, marked as answer]
Ok, I was able to find a solution to this issue. In targets: [ .target( exclude: , add "Info.plist" to the array, this defaults to the top level of the package folder, where the offending file is. Then, restart Xcode. Xcode didn't register the changes until it was restarted, even though it was showing the edit before the restart. Rebuild, et voila. Hopefully, this helps someone else.

**Reply by guoyingtao** [reply]
Thanks for the report. I will fix this issue in the next release.
Currently, you can use the master branch for Mantis Swift Package. I added "Info.plist" to the exclude array as you mentioned above.
